### PR TITLE
Add status pending agents

### LINKED ISSUE #3067: Missing `pending` option for field status in Agents overview
|Version|
|---:|
|4.x| 

**Description**
The search bar in Agents overview is not showing `pending` as an option for the field `status` 

**Additional context**
image

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team this PR resolves:
- Missing pending option for field status in Agents overview

Closes #3067 

Captura de pantalla 2021-03-23 a las 11 29 08
Captura de pantalla 2021-03-23 a las 11 19 39
Captura de pantalla 2021-03-23 a las 11 19 13

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4105
+
+### Added
+
+- Pending status suggestion for agents [#3095](https://github.com/wazuh/wazuh-kibana-app/pull/3095)
+
 ## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4104
```
Comment (Desvelao) [inline CHANGELOG.md:11]: This will need to be updated to `4.2.0-4201`.

**Comment by Desvelao** [comment]
Should we add the suggestions alphabetically sorted?
image

**Comment by pablomarga** [comment]
> Should we add the suggestions alphabetically sorted?
> image

It is already alphabetically sorted

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**